po/it/system-monitor.po

```diff
@@ -207,7 +207,7 @@ msgid "other"
 msgstr "altro"
 
 msgid "nice"
-msgstr "nice"
```

Comment:
Nice is used as detailed in this case i think, particolare or particolareggiato should do the job